sgt-back-end/index.js

```diff
@@ -0,0 +1,142 @@
+const pg = require('pg');
+const express = require('express');
+const app = express();
+const jsonMiddleware = express.json();
+
+app.use(jsonMiddleware);
+
+const db = new pg.Pool({
+  connectionString: 'postgres://dev:lfz@localhost/studentGradeTable'
+});
+
+app.get('/api/grades', (req, res) => {
+  const sql = `
+    select *
+    from "grades"
+    `;
+
+  db.query(sql)
+  .then(result => {
+
+    res.status(200).json(result.rows);
+  })
+  .catch(err => {
+    console.error(err);
+    res.status(500).json({
+      'error': 'An unexpected error occurred'
+    });
+  })
+})
+
+app.post('/api/grades', (req, res) => {
+  const grade = req.body;
+
+  if (grade.name === undefined || grade.score === undefined ||
```

Comment:
If you are just checking if something exists you can just use the bang operator. For example:

```JAVASCRIPT
if (!grade.name || !grade.score || !grade.course)
```